pkg/resources/resources_manager.go

```diff
@@ -150,6 +155,23 @@ func (rm *resourceManager) ValidateConfigs() error {
 	return nil
 }
 
+// EnsureRdmaSystemMode ensure RDMA subsystem network namespace mode is shared
+func (rm *resourceManager) EnsureRdmaSystemMode() error {
```

Comment:
so this dupliecate from rdma-cni https://github.com/Mellanox/rdma-cni/blob/b013003eae5344065fa47699aecde4a733234faa/cmd/rdma/main.go#L62-L73
right?
we can move it in https://github.com/Mellanox/rdmamap. we should have one libiray to share common code. 
WDYT?

Reply:
I didn't see these projects.  It's a good idea to move common logic onto the shared library. Some refactoring is required for this but it shouldn't introduce big changes

Reply:
my 2-cent:

netlink provides and API to get/set this configuration.
rdma_map currently only performs various get operations (mappings, stats)

i dont think there is need to move this, it is fairly simple and short.
also we would need to drop the warning log or return it as arg.